sentry/src/main/java/io/sentry/connection/HttpConnection.java

```diff
@@ -39,9 +39,14 @@
      */
     private static final String SENTRY_AUTH = "X-Sentry-Auth";
     /**
-     * Default timeout of an HTTP connection to Sentry.
+     * Default connection timeout of an HTTP connection to Sentry.
      */
-    private static final int DEFAULT_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(1);
+    private static final int DEFAULT_CONNECTION_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(1);
+    /**
+     * Default read timeout of an HTTP connection to Sentry.
+     */
+    private static final int DEFAULT_READ_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(1);
```

Comment:
I think even 1 second will be very uncommon but since people use us from all over the world and might have network blips here and there I'd prefer to default to like... 5 seconds? That's still way better than indefinite. :smile: